back-end/src/helpers/connect-to-mongo.ts

```diff
@@ -2,7 +2,7 @@ import mongoose from 'mongoose';
 import config from 'config';
 
 export async function connectToMongo() {
-  const baseUrl = process.env.CI_RUNNER ? 'mongodb://mongo' : 'mongodb://127.0.0.1';
+  const baseUrl = (process.env.CI_RUNNER ? 'mongodb://mongo' : config.has('DB_HOST'))
```

Comment:
should be `config.get`

Reply:
also unnecessary to have parenthesis around the whole statement

Reply:
and end with a semicolon (install ts-lint extension if you're using vscode)

Reply:
Sorry, 'bout that oversight. I use the parentheses for readability when dealing with multiple ? statements.